src/helpers.js

```diff
@@ -40,10 +40,14 @@ export const getEstimate = (data) => {
   let prevEstimate = undefined;
   if (estimated_next_purchase) prevEstimate = estimated_next_purchase;
 
-  const daysSinceLastTransaction = calcDaysSince(
+  let daysSinceLastTransaction = calcDaysSince(
     last_purchased_date || date_added,
   );
 
+  // If no days have passed, return the user's original guess
+  if (daysSinceLastTransaction === 0)
+    daysSinceLastTransaction = estimated_next_purchase;
```

Comment:
I'm not following this logic. Why would you want to set `daysSinceLastTransaction` to `estimated_next_purchase`?

Reply:
You answered this on the call, but an idea to make this clearer in the future is to just return `estimated_next_purchase` instead of setting `daysSinceLastTransaction` because that part is confusing.